tests/test_training/test_opt_lab_jax.py

```diff
@@ -55,8 +61,8 @@ def test_S2gate_coincidence_prob(self, n):
         def cost_fn(S):
             return -(math.abs(S.fock_array((n + 1, n + 1))[n, n]) ** 2)
 
-        opt = OptimizerJax(learning_rate=0.01)
-        opt.minimize(cost_fn, by_optimizing=[S], max_steps=300)
+        opt = OptimizerJax(euclidean_lr=0.01)
+        (S,) = opt.minimize(cost_fn, by_optimizing=[S], max_steps=300)
```

Comment:
The idea with this returning `(S,)` is that you've made explicit that it changes `S`, whereas before it was a, perhaps hidden, side effect?